source/chapters/microphones.rst

```diff
@@ -411,3 +411,28 @@ input.
 
 DJ controllers typically do not supply phantom power required for condenser
 microphones. Stand-alone audio interfaces typically do supply phantom power.
+
+Microphone Ducking
+------------------
+Microphone ducking is that effect you hear when you speak through the microphone, and the music volume is reduced by the presence of the voice signal coming through the microphone.
+While using Mixxx with a microphone, you might want the music to automatically fade into the background each time you speak, or you might prefer lowering the volume of the music to a specific level. Microphone ducking helps you achieve this.  The music will 'duck' out of the way of the microphone audio.
+
+.. figure:: ../_static/microphone_ducking.png
+   :align: center
+   :width: 40%
+   :figwidth: 90%
+   :alt: The microphone ducking section
+   :figclass: pretty-figures
+
+   The microphone ducking controls
+
+The Strength knob adjusts the microphone talkover ducking strength and its behaviour depends on any of the Microphone talkover modes described below.
```

Comment:
```suggestion
The :guilabel:`Strength` knob adjusts the microphone talkover ducking strength and its behaviour depends on any of the Microphone talkover modes described below.
```

Reply:
@deborahtrez , you can incorporate jan's changes by selecting the "commit suggestion" button, or adding it to a batch.

Reply:
Okay. I will start using this method. Thank you Owen.